features/support/pages/guidance_page.rb

```diff
@@ -0,0 +1,6 @@
+class GuidancePage < SitePrism::Page
+  set_url '/guidance/new'
+
+  element :name, 'input.t-name'
+  element :submit, 'input[type="submit"]'
```

Comment:
We mixing our approach here. We're using test CSS classes and CSS attribute selectors. Why not `input.t-submit`?